assets/scripts/fe/src/app/commons/services/interceptors/token.service.ts

```diff
@@ -34,6 +34,9 @@ export class TokenService {
 
     return n.handle(req).pipe(tap(
       resp => {
+        // Check every request if the user is authenticated, otherwise the user is redirected to the login page, 
+        // passing a parameter on the last page he/she visited.
+        this.auth.redirectUser(this.currentUrl); 
```

Comment:
huh? did you read the code base? what will happen to the requests to the backend that doesn't need the user to be authenticated? eg. `slack_url`?